redfish-core/lib/assembly.hpp

```diff
@@ -51,6 +51,44 @@ inline void
         tempyArray.at(assemblyIndex)["Status"]["Health"] = "OK";
         tempyArray.at(assemblyIndex)["Status"]["State"] = "Enabled";
 
+        // Handle special case for tod_battery assembly OEM ReadyToRemove property
+        // NOTE: The following method for the special case of the tod_battery
+        // ReadyToRemove property only works when there is only ONE adcsensor
+        // handled by the adcsensor application.
+        if (sdbusplus::message::object_path(assembly).filename() == "EBMC")
+        {
+            auto bus = sdbusplus::bus::new_default();
```

Comment:
Please don't do this. 
Please use             crow::connections::systemBus->async_method_call( 

https://github.com/ibm-openbmc/bmcweb/blob/1020/redfish-core/lib/systems.hpp#L1088

There is 100s of examples in bmcweb